runtime/src/accounts_db.rs

```diff
@@ -1036,21 +1086,7 @@ impl AccountsDB {
         update_index.stop();
         trace!("reclaim: {}", reclaims.len());
 
-        let mut remove_dead_accounts = Measure::start("store::remove_dead");
```

Comment:
just to be sure; these `Measure`s are no longer needed? The code move to `handle_reclaims` didn't carry with them.

Reply:
They are not strictly necessary. They are just adding some timing information when you run with the profiler on.